lib/language_variants_filter.js

```diff
@@ -86,8 +88,20 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 // TODO: Eventually we want to think of a better way to support
                 // other content pure-fetching, but right now let's use a magic
                 // header.
-                req.headers['cache-control'] = 'no-cache,no-store';
-                return next(hyper, req);
+                mReq.headers['cache-control'] = 'no-cache,no-store';
+                
+                //
+                return next(hyper, mReq)
+                .then((res) => mwUtil.canConvertLangVariant(hyper, mReq, revision.page_language)
```

Comment:
This is not needed. If `shouldConvert` returns `true`, then so will `canConvert`. Since at this point in the code we are sure `shouldConvert` returned `true`, no need to check this again.

Reply:
Done.